security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -48,6 +53,11 @@ var (
 	newFileWatcher = filewatcher.NewWatcher
 	// The total timeout for any credential retrieval process, default value of 10s is used.
 	totalTimeout = time.Second * 10
+
+	// SupportRotationWithCert is a field controlling the use the cert for rotation
+	// If is set to "true", we will use the old cert to rotate instead of token
+	SupportRotationWithCert = env.RegisterStringVar("CERT_ROTATE", "",
```

Comment:
Make the name of the variable and the env var same, the help text also need some update to make it more clear about what does it mean of `use cert, ...`.

Reply:
Will refactor the Name. Because currently default we use the token for CSR validation. And with this variable we can allow the user to choose the rotation way instead of token(which is more suggestable based on the talk with howard and limin)